### ISSUE TITLE
browser autocomplete does not feed the json editor

### ISSUE THREAD

**Issue by shauligal** [jdorn/json-editor#609]
When the browser (Safari) offers to auto-complete a text in an input box, the selected value does not arrive to the JsonEditor, and its state is left empty even though the input box shows up "full" on the screen. I realize that it's a generic javascript issue: autocomplete does not trigger certain events.
I found the following discussion here:
https://github.com/jdorn/json-editor/issues/140
but it did not help resolve it.
any ideas?

**Comment by m-a-choquette** [comment]
Hi @shauligal ,
have you found a workaround for this ?

**Comment by shauligal** [comment]
No. I'm just avoiding the auto-complete pop-up and typing the whole value. It would really be nice to solve this properly.
